.github/blunderbuss.yml

```
@@ -0,0 +1,5 @@
+assign_issues:
+  - googleapis/langchain-alloydb
+ignore_authors:
+  - dependabot
+  - renovate
```

nit: add newline at end of file
```suggestion
  - renovate
  
```